apps/netbox/ptlsbox/netbox-redis.enc.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+data:
+  postgresql-password: ENC[AES256_GCM,data:G++joyaK/GT2sHyxrE3LtyYgxkATf3vFzWm1eA==,iv:KqUYsCI1EhaY3n0VwznbbGswDVg2K9jbp6m6OSNjtFI=,tag:uycmFiPw1B1mM7Fgn+sG9g==,type:str]
+kind: Secret
+metadata:
+  creationTimestamp: null
+  name: netbox-postgresql
```

Comment:
this right? not netbox-redis?

Reply:
oo let me check that

Reply:
fixed, not sure how both secrets ended up in there